### PR TITLE
feat: add <space> as an action for gestures

### PR DESCRIPTION
**Pull request by tomsaleeba**
I'm trying to move to AnySoft from Minuum, which has this feature, and I can't live without it.

It works (I'm using it) but I'm not an Android dev so hopefully I've done it right.

### CODE REVIEW

File: ime/app/src/main/java/com/anysoftkeyboard/AnySoftKeyboard.java [hunk 1]
```diff
@@ -444,6 +444,9 @@ private void onFunctionKey(final int primaryCode, final Key key, final boolean f
                     sendDownUpKeyEvents(keyEventKeyCode);
                 }
                 break;
+            case KeyCodes.SPACE:
```
Comment (menny) [inline ime/app/src/main/java/com/anysoftkeyboard/AnySoftKeyboard.java:447]: I don't think this code can ever execute. The method `onFunctionKey` is only called if the key-code is a negative value (which SPACE is not).

Comment (tomsaleeba) [reply ime/app/src/main/java/com/anysoftkeyboard/AnySoftKeyboard.java]: Oh, we don't need that. I was just following other examples in the code and making a copy for "space". I've removed it.

**Comment by menny** [comment]
Use `./gradlew googleJavaFormat` to fix the code styling issue

File: ime/app/src/test/java/com/anysoftkeyboard/ime/AnySoftKeyboardKeyboardSubtypeTest.java [hunk 2]
```diff
@@ -79,7 +79,8 @@ public void testAvailableSubtypesReported() {
                     1618259652,
                     -517805346,
                     -1601329810,
-                    -1835196376
+                    -1835196376,
```
Comment (menny) [inline ime/app/src/test/java/com/anysoftkeyboard/ime/AnySoftKeyboardKeyboardSubtypeTest.java:82]: Why did you make this change?

File: ime/app/src/test/java/com/anysoftkeyboard/keyboards/KeyboardAddOnTest.java [hunk 3]
```diff
@@ -51,7 +51,7 @@ public void testGetEnabledDefaultFromAllKeyboards() throws Exception {
                     addOnAndBuilder.getId(), addOnAndBuilder.getKeyboardDefaultEnabled());
         }
 
-        Assert.assertEquals(10, keyboardsEnabled.size());
+        Assert.assertEquals(11, keyboardsEnabled.size());
```
Comment (menny) [inline ime/app/src/test/java/com/anysoftkeyboard/keyboards/KeyboardAddOnTest.java:54]: And this?

File: ime/app/src/test/java/com/anysoftkeyboard/ime/AnySoftKeyboardKeyboardSubtypeTest.java (hunk 2 above)
Comment (tomsaleeba) [reply ime/app/src/test/java/com/anysoftkeyboard/ime/AnySoftKeyboardKeyboardSubtypeTest.java:82]: Sorry, sloppy workspace management :grimacing: . I'm adding a new layout on another branch and got things mixed up.

I've removed that stuff and done a rebase and squash on the latest master to tidy it all up.

File: ime/app/src/test/java/com/anysoftkeyboard/keyboards/KeyboardAddOnTest.java (hunk 3 above)
Comment (tomsaleeba) [reply ime/app/src/test/java/com/anysoftkeyboard/keyboards/KeyboardAddOnTest.java:54]: See previous comment

**Comment by tomsaleeba** [comment]
That failed check seems to be from master, not from my changes. I can rebase to the previous commit, which had successful CI, if you like?

**Comment by menny** [comment]
I've unbroken master. Thanks for letting me know.

Please rebase this branch.

**Comment by tomsaleeba** [comment]
Rebased and succesfully built :smiley:

**Pull request merged**